src/app/components/quiz.tsx

```diff
@@ -0,0 +1,110 @@
+"use client";
+
+import { useState } from "react";
+import clsx from "clsx";
+
+export interface QuizParams {
+  className?: string;
```

Comment:
className: string | undefined

use this just for boolean:
someBoolean?: boolean